docs/develop/scripting-fundamentals.md

```diff
@@ -191,15 +197,65 @@ function main(workbook: ExcelScript.Workbook) {
     // Get the first worksheet.
     let sheet = workbook.getWorksheets()[0];
 
-    // Add a table that uses the data in C3:G10.
+    // Add a table that uses the data in A1:G10.
     let table = sheet.addTable(
-      "C3:G10",
+      "A1:G10",
        true /* True because the table has headers. */
     );
+    
+    // Give the table a name for easy reference in other scripts.
+    table.setName("MyTable");
+}
+```
+
+> [!TIP]
+> Most Excel objects have a `setName` method. This gives you an easy way to access Excel objects later in the script or in other scripts for the same workbook.
+
+#### Check if an object exists in the collection
+
+Scripts often need to check if a table or similar object exists before continuing. You can use the names given by scripts or through the Excel UI to identify necessary objects and act accordingly. `get` methods return `undefined` when the requested object is not in the collection.
+
+The following script requests a table named "MyTable" and uses an `if/else` statement to check if the table was found.
```

Comment:
Mozilla formats this as [if...else](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else), like try...catch. 
```suggestion
The following script requests a table named "MyTable" and uses an `if...else` statement to check if the table was found.
```